apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.js

```diff
@@ -16,12 +16,31 @@ const AcceptOrRejectOrganizationInviteService = new GQLCustomSchema('AcceptOrRej
             resolver: async (parent, args, context, info, extra = {}) => {
                 if (!context.authedItem.id) throw new Error('[error] User is not authenticated')
                 const { id, data } = args
+                const authedItem = context.authedItem
                 let { isRejected, isAccepted, ...restData } = data
                 isRejected = isRejected || false
                 isAccepted = isAccepted || false
 
-                const obj = await updateOrganizationEmployee(context, id, { isRejected, isAccepted, ...restData })
-                return await getById('OrganizationEmployee', obj.id)
+                let employee
```

Comment:
it's better to move this logic inside Invite mutation.

Reply:
+1

Reply:
@pahaz task need to be descussed